src/class_singleWorker.py

```diff
@@ -546,7 +546,7 @@ def sendBroadcast(self):
                 # , privEncryptionKeyHex
                 privSigningKeyHex, _, pubSigningKey, pubEncryptionKey = \
                     self._getKeysForAddress(fromaddress)
-            except:
+            except:  # noqa:E722
```

Comment:
replace with `KeyError`

add general error handler that logs warning

Reply:
Updated